docs/modules/data-structures/pages/reliable-topic.adoc

```diff
@@ -175,4 +175,38 @@ messages when they are added or removed.
 tries to read in batches. Its default value is 10.
 * `topic-overload-policy`: Policy to handle an overloaded topic.
 Available values are `DISCARD_OLDEST`, `DISCARD_NEWEST`, `BLOCK` and `ERROR`.
-Its default value is `BLOCK`. See <<slow-consumers, Slow Consumers>> for definitions of these policies.
\ No newline at end of file
+Its default value is `BLOCK`. See <<slow-consumers, Slow Consumers>> for definitions of these policies.
+
+[[reliable-message-listener]]
+== ReliableMessageListener
+
+ReliableMessageListener is a MessageListener to better integrate with the reliable topic.
+
+If a regular MessageListener is registered on a reliable topic, the message listener works fine, but it can't do much more than listen to messages.
+If a ReliableMessageListener is registered on a normal topic, only the MessageListener methods are called.
+
+**Durable Subscription**
+
+The ReliableMessageListener allows you to control where you want to start processing a message when the listener is registered. This makes it possible to create a durable subscription by storing the sequence of the last message and using this sequenceId as the sequenceId to start from.
+
+**Exception handling**
+
+The ReliableMessageListener also gives the ability to deal with exceptions using the `isTerminal(Throwable)` method. If a plain MessageListener is used, then it won't terminate on exceptions and it will keep on running. But in some cases it is better to stop running.
```

Comment:
I think this can be explained more in the documentation (or we can update the javadoc as well)

```suggestion
The ReliableMessageListener also gives the ability to deal with exceptions using the `isTerminal(Throwable)` method. This method allows you to control which exceptions should terminate the execution of the listener and cancel it. If a plain MessageListener is used, it won't terminate on exceptions and it will keep on running. But in some cases it is better to stop running.
```